cmd/vela-cli/main.go

```diff
@@ -153,23 +153,26 @@ func main() {
 
 // load is a helper function that loads the necessary configuration for the CLI.
 func load(c *cli.Context) error {
-	config := c.String("config")
+	// create the config file configuration
+	//
+	// https://pkg.go.dev/github.com/go-vela/cli/action/config?tab=doc#Config
+	conf := &config.Config{
+		Action: "load",
+		File:   c.String("config"),
+	}
 
-	_, err := os.Stat(config)
+	// validate config file configuration
+	//
+	// https://pkg.go.dev/github.com/go-vela/cli/action/config?tab=doc#Config.Validate
+	err := conf.Validate()
 	if err == nil {
-		err = altsrc.InitInputSourceWithContext(c.App.Flags, func(context *cli.Context) (altsrc.InputSourceContext, error) {
-			yaml, err := altsrc.NewYamlSourceFromFile(config)
-			return yaml, err
-		})(c)
+		// execute the load call for the config file configuration
+		//
+		// https://pkg.go.dev/github.com/go-vela/cli/action/config?tab=doc#Config.Load
+		err = conf.Load(c)
```

Comment:
This is where we replace the old way of loading the config file for our new function